docs/sessions.md

```diff
@@ -297,5 +332,6 @@ For complete documentation, please see the [Session API docs](api/sessions.md).
 | [`write_table`][daft.Session.write_table]                                       | Writes a dataframe to the table.                                   |
 | [`set_catalog`][daft.Session.set_catalog]                                       | Sets the current catalog.                                          |
 | [`set_namespace`][daft.Session.set_namespace]                                   | Sets the current namespace.                                        |
+| [`set_provider`][daft.Session.set_provider]                                     | Sets the current provider.                                         |
```

Comment:
There are additional session methods as well that I believe are missing like attach_session.